README.md

```diff
@@ -1,34 +1,54 @@
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+# Next.js Starter ButterCMS
 
-## Getting Started
+Next.js starter project using ButterCMS. The project uses the sample content that is automatically created when you
+create your free trial account on ButterCMS.
 
-First, run the development server:
+[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/clone?repository-url=https%3A%2F%2Fgithub.com%2FButterCMS%2Fnextjs-starter-buttercms)
 
+- Demo: https://nextjs-starter-buttercms.vercel.app/
+
+## Installation
+
+First, install the dependencies by running this command:
 ```bash
-npm run dev
+npm install
 # or
-yarn dev
+yarn install
 ```
+### Set API Token
 
-Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
-
-You can start editing the page by modifying `pages/index.js`. The page auto-updates as you edit the file.
+To fetch your ButterCMS content, add your API token as an environment variable. Set `NEXT_PUBLIC_BUTTER_CMS_API_KEY=<Your API Token>` in `.env`. Get [your free API token](https://buttercms.com/join/).
 
-[API routes](https://nextjs.org/docs/api-routes/introduction) can be accessed on [http://localhost:3000/api/hello](http://localhost:3000/api/hello). This endpoint can be edited in `pages/api/hello.js`.
+```bash
+$ echo 'NEXT_PUBLIC_BUTTER_CMS_API_KEY=your_token' >> .env
+```
 
-The `pages/api` directory is mapped to `/api/*`. Files in this directory are treated as [API routes](https://nextjs.org/docs/api-routes/introduction) instead of React pages.
+### Run the local server
+All that's left is running development server
 
-## Learn More
+```bash
+npm run dev
+# or
+yarn dev
+```
+Your starter is live: [http://localhost:3000](http://localhost:3000)!
+#### Building the app
```

Comment:
Same thing as the django project, i'm wondering about something like this: https://github.com/ButterCMS/django-starter-buttercms/pull/1#discussion_r755201757 where we give the user a way of knowing if the installation has proceeded successfully and server is running? What do you think?

Reply:
I like the idea, so I only slightly rephrased your text to better fit with the next.js command